app/tools/formTool.js

```diff
@@ -0,0 +1,37 @@
+function formBuilder(...args) {
+  let form = '';
+  // check for data objects
+  if (args.length > 0) {
+    let i;
+    let x;
+    // through data create specified input type
+    for (i = 0; i < args.length; i += 1) {
+      if (args[i].field === 'input') {
+        form += '<label for="' + args[i].id + '" class="label">' + args[i].label + '</label>';
```

Comment:
Very good use of the ES6 args usage to efficiently handle the form items but what if no label was defined? Should there not be  conditional logic to resolve that? Especially when your documentation makes no mention of explicitly needing the label defined.